lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -61,6 +61,16 @@
         ini:
           - {key: host_key_auto_add, section: paramiko_connection}
         type: boolean
+      gss_auth:
+        default: False
```

Comment:
Would it not be convenient to have `gss_auth` set by default to True, as using ssh it is already set.

Reply:
That would be a behavior change.